### PR TITLE
Grid packing algorithm

### PR DESCRIPTION
**Pull request by liquidx**
A dumb algorithm to do grid packing of random sizes. 

emoji-ffe7bd-1680x1050@2x (2)

@alcor @dsandler

### CODE REVIEW

**Comment by dsandler** [comment]
aww yeah, I love this!

**Comment by dsandler** [comment]
a bit like [DessertCase](https://youtu.be/Bk5Jh9kWKPQ?t=83) but this has a more flexible grid

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
 # emoji-supply
 
 Code for http://emoji.supply/wallpaper
+
+# Dev
+
+```
+python3 -m http.server 8989 --directory docs
```
Comment (dsandler) [inline README.md:8]: or just open docs/wallpaper/index.html in Chrome directly, that's what I do

**Review by dsandler — COMMENTED** [review]
LGTM++

File: README.md (hunk 1 above)
Comment (liquidx) [reply README.md:8]: Oh ... that's so obvious.

**Pull request merged**